core/templates/components/summary-tile/subtopic-summary-tile.component.html

```diff
@@ -2,11 +2,11 @@
      (click)="openSubtopicPage()"
      [ngClass]="{'disabled-link': !classroomUrlFragment}">
   <div class="subtopic-thumbnail protractor-test-subtopic-thumbnail"
-       [ngStyle]="{background: thumbnailBgColor}">
+       [ngStyle]="{background: subtopic.getThumbnailBgColor()}">
```

Comment:
Why this change? I think the previous way is preferred more during migration for the proper initialization of components?

Reply:
Accepting your suggestion.